packages/launchpad/src/setup-actions.ts

```diff
@@ -22,7 +22,11 @@ export const setUpActionsForTasks = ( {
 	uiContext = 'calypso',
 }: LaunchpadTaskActionsProps ): Task[] => {
 	const { recordTracksEvent, checklistSlug, tasklistCompleted, launchpadContext } = tracksData;
-	const { setShareSiteModalIsOpen, siteLaunched, setActiveChecklist } = extraActions;
+	const {
+		setShareSiteModalIsOpen = undefined,
+		siteLaunched = undefined,
+		setActiveChecklist,
+	} = extraActions;
```

Comment:
I don't think we should keep this change. But it's a temp fix so that I can load Launchpad without error. Without this, I get a 'cannot destructure property that is undefined' error due to setShareSiteModalIsOpen and siteLaunchped being optional props that I'm not providing here to this method. 

I need to discuss with Launchpad team how they want to handle this. See thread here: p1695931203171119/1695928659.435109-slack-C057AH42XQD

Reply:
It may be worth adding this as defensive code in a dedicated PR, which should be pretty quick to ship.